test/functionalTest/cases/787_cprs_full_functional/two_attributes_in_one_cpr_three_in_another.test

```diff
@@ -0,0 +1,624 @@
+    "registrationId": "REGEX([0-9a-f]{24})"
+}
+
+
+07. Register E1/T1/A2 in CB, provApp: CP1
+=========================================
+HTTP/1.1 200 OK
+Content-Length: 74
+Content-Type: application/json
+Date: REGEX(.*)
+
+{
+    "duration": "P1M", 
+    "registrationId": "REGEX([0-9a-f]{24})"
+}
+
+
+08. Register E1/T1/A3 in CB, provApp: CP1
+=========================================
+HTTP/1.1 200 OK
+Content-Length: 74
+Content-Type: application/json
+Date: REGEX(.*)
+
+{
+    "duration": "P1M", 
+    "registrationId": "REGEX([0-9a-f]{24})"
+}
+
+
+09. Register E1/T1/A4 in CB, provApp: CP2
+=========================================
+HTTP/1.1 200 OK
+Content-Length: 74
+Content-Type: application/json
+Date: REGEX(.*)
+
+{
+    "duration": "P1M", 
+    "registrationId": "REGEX([0-9a-f]{24})"
+}
+
+
+10. Register E1/T1/A5 in CB, provApp: CP2
+=========================================
+HTTP/1.1 200 OK
+Content-Length: 74
+Content-Type: application/json
+Date: REGEX(.*)
+
+{
+    "duration": "P1M", 
+    "registrationId": "REGEX([0-9a-f]{24})"
+}
+
+
+11. Query E1/T1/A1+A2+A3+A4+A5 (A1-A3 in CP1, A4-A5 in CP2)
+===========================================================
+HTTP/1.1 200 OK
+Content-Length: 1103
+Content-Type: application/json
+Date: REGEX(.*)
+
+{
+    "contextResponses": [
+        {
+            "contextElement": {
+                "attributes": [
+                    {
+                        "name": "A1", 
+                        "type": "string", 
+                        "value": "A1 in CP1"
+                    }, 
+                    {
+                        "name": "A2", 
+                        "type": "string", 
+                        "value": "A2 in CP1"
+                    }, 
+                    {
+                        "name": "A3", 
+                        "type": "string", 
+                        "value": "A3 in CP1"
+                    }
+                ], 
+                "id": "E1", 
+                "isPattern": "false", 
+                "type": "T1"
+            }, 
+            "statusCode": {
+                "code": "200", 
+                "reasonPhrase": "OK"
+            }
+        }, 
+        {
+            "contextElement": {
+                "attributes": [
+                    {
+                        "name": "A4", 
+                        "type": "string", 
+                        "value": "A4 in CP2"
+                    }, 
+                    {
+                        "name": "A5", 
+                        "type": "string", 
+                        "value": "A5 in CP2"
+                    }
+                ], 
+                "id": "E1", 
+                "isPattern": "false", 
+                "type": "T1"
+            }, 
+            "statusCode": {
+                "code": "200", 
+                "reasonPhrase": "OK"
+            }
+        }
+    ]
+}
+
+
+--TEARDOWN--
+brokerStop CB
+brokerStop CP1
+brokerStop CP2
+dbDrop CB
+dbDrop CP1
+dbDrop CP2
```

Comment:
The existing test are pretty good... the cover a good range os cases.

However, I miss the following scenarios:

* Query with empty attributes with attributes in CPrs, e.g. query E-none and get E-A1,A2,A3,A4 (A1-A2 comming from one CPr and A3-A4 comming from another CPrs).
* Query with empty attributes "mixing" attributes in CPrs and CB, e.g. query E-[none] and get E-A1,A2,A3 (A1 comming from CB and A2-A3 comming from a CPr).
* Situations in which CPr doesn't hold the data, although it is registered in that way. Eg. query E-A, being E-A register in CPr, but the  CPr is empty. It should return 404 to the client.
* Similar to the former, but with pieces of data that preclude to return a 404 to the client. Eg. query E-A1,A2, being E-A1 register in CPr, but the  CPr is empty. E1-A2 is locally stored in CB. The CPr will return 404 for the E-A1 forward, but given that E1-A2 has been resolved locally, the client will finally receive E-A2 (not 404)
* Variation of the former in which E1-A2 is in a second CPr instead that in CB.
* Cases in which local resolution "shadows" CPr. E.g. E-A1 is registered in CPr and at the same time stored locally in the CB. A query for E-A1 will return the value in the CB, no tin the CPr.